base/src/main/java/io/customer/base/extenstions/DateExtensions.kt

```diff
@@ -1,8 +1,26 @@
 package io.customer.base.extenstions
 
+import java.text.SimpleDateFormat
 import java.util.*
 import java.util.concurrent.TimeUnit
 
+enum class DateFormat {
```

Comment:
maybe `DateFormat` should be in `utils` folder rather than an extension class? to follow the conventions also, If someone was looking for something similar. it would be hard to guess it would be in extension folder